docs/manual-EN/1.overview/3.design-and-architecture/2.storage-design.md

```diff
@@ -34,15 +34,15 @@ Pict.2 Vertex Key Format
 
 ---
 
-- `Type`: one byte, used to indicate key type, current types are data, index, system, etc.
-- `Part ID`: three bytes, used to indicate the sharding partition. This field can be used to **scan the partition data based on the prefix when re-balancing the partition**
-- `Vertex ID`: eight bytes, used to indicate vertex ID
-- `Tag ID`: four bytes, used to relate tag
-- `Timestamp`: eight bytes, not available to users, used in MVCC in the future
+- `Type`: one byte, to indicate the key type. e.g. data, index, system, etc.
+- `Part ID`: three bytes, used to indicate the (sharding) partition id. It's designed for the data migration/balance operation by **prefix-scanning all the data in a partition.**
+- `Vertex ID`: eight bytes, used to indicate vertex ID. Two nodes with an identity vertexID are considered as the same node.
+- `Tag ID`: four bytes, used to indicate its tag's (encoded) ID.
+- `Timestamp`: eight bytes, not visible to users. Reserved for `Multiversion concurrency control (MVCC)`.
 
-Each edge in **Nebula Graph** is modeled as two independent key-values. One is stored in the same partition as the source vertex, namely out-edge. The other one is stored in the same partition as the destination vertex, namely in-edge. Generally, out-key and in-key are stored in different partitions.
+Each edge in **Nebula Graph** is modeled and stored as two independent key-values. One, namely the `out-edge`,  is stored in the same partition as the `source vertex`. The other one, namely `in-edge`, is stored in the same partition as the `destination vertex`. So generally, out-key and in-key are in different partitions.
 
-There may be multiple edge types or multiple edges with the same edge type between two vertices. For example, define an edge type 'transfer', user A may transfer money to user B multiple times, thus a field `rank` is added to distinguish the transfer records between the two. Edge key format is shown in Picture Three:
+Between two vertices, edges with the same type are accepted, and different types are legal as well. For example, by define an edge type '`money-transfer-to`', `user A` can transfer money to `user B` at two timestamps. Thus a field, namely `rank`, is added to (to the key part of the timestamp) to distinguish which transfer records is referring.  Edge key format is shown in Picture Three:
```

Comment:
accepted > allowed
by defining
is added to (to the key part of the timestamp) distinguish 
Picture > Figure